python-sdk/tutorials/automl-with-azureml/image-classification-multiclass/auto-ml-image-classification-multiclass.ipynb

```diff
@@ -715,13 +810,102 @@
     "\n",
     "plt.show()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Generate Explanations\n",
```

Comment:
just thinking out loud, if the endpoint can take a super long time to return in some circumstances for explainability (like > 60 seconds or something if the user adopts it as a main way of generating explanations), I wonder if we should expose it to the user or not / whether or not we should only expose batch scoring. just thinking out loud / not sure -- maybe a decision can be made by others